chain-storage/src/store.rs

```diff
@@ -298,3 +299,270 @@ fn compute_fast_link(depth: u64) -> u64 {
         0
     }
 }
+
+//#[cfg(test)]
```

Comment:
Why is this now enabled in non-test configurations?
If substantial additional modules are needed to support the tests, it's better to put them in subdirectories under `tests` in the crate directory, and organize the test code into standalone tests.